frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts

```diff
@@ -451,11 +450,14 @@ export const sessionRecordingDataLogic = kea<sessionRecordingDataLogicType>([
             null as SessionPlayerSnapshotData | null,
             {
                 pollRecordingSnapshots: async (_, breakpoint: BreakPointFunction) => {
+                    const params = { ...REALTIME_POLLING_PARAMS }
+
+                    if (values.featureFlags[FEATURE_FLAGS.SESSION_REPLAY_V3_INGESTION_PLAYBACK]) {
+                        params.version = '3'
```

Comment:
we should include this param in the reported timings from here so we can validate if it's faster/slower